README.md

```diff
@@ -0,0 +1,144 @@
+
+## Usage
+1. Setup your *WherebyRoomFragment* object in your source file and layout:
+```
+WherebyRoomFragment roomFragment = new WherebyRoomFragment();
+```
+```
+<FrameLayout  
+  android:id="@+id/layout_whereby_fragment_container"  
+  .../>
+  ```
+
+2. Get/create the room URL
+```
+URL roomURL = new URL("https://...");
+```
+
+3. Create a *WherebyRoom* that contains the configuration to passed to the fragment
+```
+WherebyRoom room = new WherebyRoom(roomURL);
+```
+
+4. Add optional room parameters
+```
+WherebyRoomParameters roomParameters = new WherebyRoomParameters(); 
+roomParameters.setCameraEnabledAtStart(true);  
+roomParameters.setMicrophoneEnabledAtStart(true); 
+roomParameters.setDisplayName("Participant name");  
+// ...
+room.setParameters(roomParameters);
+```
+
+5. Receive asynchronous events during the meeting
+
+There are two ways to receive asynchronous events once you have joined the meeting:
+- Setting up an event listener on the *roomFragment*: this option provides a list of optional callback methods that you can implement.
+```
+roomFragment.setEventListener(new WherebyEventListener() {  
+
+    @Override  
+    public void onMicrophoneToggled(boolean enabled) {  
+        // Update custom microphone button, for instance
+    }  
+  
+    @Override  
+    public void onCameraToggled(boolean enabled) {  
+        // Update custom camera button, for instance
+    }  
+
+    // ...
+});  
+```
+- Registering a broadcast receiver: this option allows to receive a *WherebyRoomEvent* anywhere, including in a service.
+```
+room.setEventBroadcastEnabled(true);
+IntentFilter intentFilter = new IntentFilter(ROOM_BROADCAST_EVENT_ACTION);  
+BroadcastReceiver broadcastReceiver = new BroadcastReceiver() {
+    @Override  
+    public void onReceive(Context context, Intent intent) {  
+        WherebyRoomEvent event = (WherebyRoomEvent) intent.getSerializableExtra(ROOM_BROADCAST_EVENT_NAME);  
+        // Process event here
+    }  
+};  
+LocalBroadcastManager.getInstance(context).registerReceiver(broadcastReceiver, intentFilter);
+```
+*Remember to unregister the broadcast receiver when leaving the meeting:*
+```
+LocalBroadcastManager.getInstance(context).unregisterReceiver(broadcastReceiver);
+```
+
+6. Send commands once the meeting has started
+```
+toggleCameraButton.setOnClickListener(view -> roomFragment.toggleCamera());  
+toggleMicrophoneButton.setOnClickListener(view -> roomFragment.toggleMicrophone());
+```
+
+7. Load the fragment with its arguments and join the meeting
+```
+Bundle bundle = new Bundle();  
+bundle.putSerializable(ROOM_ARGUMENT_KEY, room);  
+roomFragment.setArguments(bundle);  
+FragmentManager fragmentManager = getSupportFragmentManager();  
+FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();  
+fragmentTransaction.replace(frameLayout, roomFragment);  
+fragmentTransaction.commit();  
+roomFragment.join();
+```
+
+8. Remove the fragment
```

Comment:
Suggest to clarify with something like "when it's no longer needed" or "after the meeting has ended". Currently it can feel like it's a next step in the setup process.